### PR TITLE
Add NoBinaryStringFixer

### PR DESCRIPTION
**Pull request by ntzm**

### CODE REVIEW

**Comment by julienfalque** [comment]
What about converting `(binary)` cast to `(string)`?

**Comment by ntzm** [comment]
Is that worth doing in the same fixer?

**Comment by julienfalque** [comment]
You're right, this should be a different fixer.

File: src/Fixer/StringNotation/NoBinaryStringFixer.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        foreach ($tokens as $index => $token) {
+            if (!$token->isGivenKind([T_CONSTANT_ENCAPSED_STRING, T_START_HEREDOC])) {
```
Comment (SpacePossum) [inline src/Fixer/StringNotation/NoBinaryStringFixer.php]: I think NOWDOC could also be fixed?

```php
<?php

echo b<<<'EOT'
test
EOT;
?>
```

please add a HEREDOC code sample as well :)

Comment (keradus) [reply src/Fixer/StringNotation/NoBinaryStringFixer.php]: it's not like i was in the middle of merging here, and then boom, comment....
merge stopped just in time

**Comment by keradus** [comment]
@ntzm , FYI - I have squashed your PR. feel free to push your original branch (with `--force`) or use squashed one

File: src/Fixer/StringNotation/NoBinaryStringFixer.php (hunk 1 above)
Comment (SpacePossum) [reply src/Fixer/StringNotation/NoBinaryStringFixer.php]: sorry about that one, you guys are just way to fast for me today ;)

Comment (ntzm) [reply src/Fixer/StringNotation/NoBinaryStringFixer.php]: NOWDOCs are dealt with here, they have a token of T_START_HEREDOC though 😆

**Comment by ntzm** [comment]
Pushed using force, added a HEREDOC code sample

File: src/Fixer/StringNotation/NoBinaryStringFixer.php (hunk 1 above)
Comment (SpacePossum) [reply src/Fixer/StringNotation/NoBinaryStringFixer.php]: what, damn PHP tokenizer haha, thanks man!

**Comment by keradus** [comment]
Thank you @ntzm.

**Pull request merged**